Generators/Generators.gradle

```diff
@@ -292,6 +292,20 @@ task generatePythonFigureWrapperAssert(type: JavaExec, dependsOn: 'classes') {
     onlyIf { System.getenv('NO_ASSERT') != 'true' }
 }
 
+task generatePythonFigureWrapper2(type: JavaExec, dependsOn: 'classes') {
+    description 'Run GeneratePythonFigureWrapper'
+    dependsOn generatePlottingConvenience
+
+    main = 'io.deephaven.plot.util.GeneratePyV2FigureAPI'
+    args devRoot, 'false'
+    classpath = sourceSets.main.runtimeClasspath
+    workingDir = workDir
+    systemProperty 'workspace', workspace
+    systemProperty 'devroot', devRoot
+    systemProperty 'Configuration.rootFile', 'dh-defaults.prop'
+    dependsOn generatePyDoc, ':Generators:generateFigureImmutable'
+}
```

Comment:
Personal opinion: it is better to have the generator run every time than mess with the assert, since the assert effectively does the same work, then blames you for not having run the generator, which _would have done the same thing without the finger wag_. That said, I'd go further and never check in the generated output unless we absolutely have to, and then always run the generate task as part of the build.

That said, this is an exec task, has no inputs or outputs declared, and yes, nothing calls it, which is going to be a problem. If the output is checked in, I could see making an assert task (and making `check` depend on that).